frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Flex, FlexItem, Form, FormGroup, Title } from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { NamespaceStoreKind } from '../../../../types';
+import { NamespaceStoreList } from '../../../namespace-store/namespace-store-table';
+import { Action, State } from '../../state';
+import NamespaceStoreModal from '../../../namespace-store/namespace-store-modal';
+import { NamespaceStoreDropdown } from '../../../namespace-store/namespace-store-dropdown';
+
+export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React.memo(
+  ({ state, dispatch, namespace }) => {
+    const { t } = useTranslation();
+    const openModal = () => NamespaceStoreModal({ namespace });
+    const [selectedCount, setSelectedCount] = React.useState(state.readNamespaceStore.length);
+    const [enabledItems, setEnabledItems] = React.useState(
+      state.readNamespaceStore.map((res) => res?.metadata.name),
+    );
+
+    React.useEffect(() => {
+      dispatch({ type: 'setReadNamespaceStore', value: [] });
+      dispatch({ type: 'setWriteNamespaceStore', value: [] });
+    }, [dispatch]);
+
+    React.useEffect(() => {
+      const readItems = state.readNamespaceStore.map((res) => res?.metadata.name);
+      setEnabledItems(readItems);
```

Comment:
```suggestion
      const readItems = state.readNamespaceStore.map(getName);
      setEnabledItems(readItems);
```
Remove this from the initial state.

Reply:
ok